.github/workflows/protocol-devchain-anvil.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
 
       - name: Generate migrations
-        run: ./migrations_sol/run_integration_tests_in_anvil.sh
+        run: ./scripts/foundry/run_integration_tests_in_anvil.sh
```

Comment:
Note-to-self: Consider refactoring this like here

https://github.com/celo-org/celo-monorepo/blob/b0fb5f9f9a2299575b5ab4c674edcfc5762546b5/.github/workflows/protocol_tests.yml#L121-L129

Reply:
This is done in https://github.com/celo-org/celo-monorepo/pull/11023/commits/c7a451a53d475e03155518fbfbc7f696219afcd0